map_gui/src/render/road.rs

```diff
@@ -28,8 +28,10 @@ impl DrawRoad {
         }
     }
 
-    pub fn render_center_line(&self, app: &dyn AppLike) -> GeomBatch {
+    pub fn render_center_line<P: AsRef<Prerender>>(&self, app: &dyn AppLike, prerender: &P) -> GeomBatch {
```

Comment:
Is it alright to pass prerender here?

Reply:
Yep, totally fine! This use of a generic is basically to support passing in `EventCtx` (used when rendering roads while initially constructing objects, or in response to a keyboard/mouse event) or `GfxCtx` (used while drawing stuff)